### PR TITLE
fix(RHINENG-10919):  Rephrase workspaces

### PR DESCRIPTION
**Pull request by gkarat**
Fixes https://issues.redhat.com/browse/RHINENG-10919.

### CODE REVIEW

**Review by adonispuente — APPROVED** [review]
LGTM

**Review by Hyperkid123 — APPROVED** [review]

File: docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml [hunk 1]
```diff
@@ -47,13 +47,13 @@ spec:
               1. **inventory:hosts:read** → Allows users to view systems (needed to view systems both inside and outside a workspace).
               1. **inventory:hosts:write** → Allows users to Rename or Delete systems.
               1. **inventory:workspaces:read** → Allows users to view workspaces, and general info (not including systems in it).
-              1. **inventory:workspace:write** → Allows users to edit inventory workspace membership (add and remove systems from workspaces).
+              1. **inventory:workspace:write** → Allows users to edit workspace membership (add and remove systems from workspaces).
         1. Select inventory permissions as needed. Here are some possible variations: <br>
             - If you want to give a user full access to the workspace and its systems → select all 4 permissions.
             - If you want to give a user full access to the systems inside a workspace → but not workspace editing access, give them inventory:hosts:read, inventory:hosts:write, and inventory:workspace:read, but NOT inventory:workspace:write.
```
Comment (computercamplove) [inline docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml:53]: i think in phase 1 platex team updating only role's display name, but not premonitions - it should be still `inventory:groups: `

Comment (gkarat) [reply docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml]: Fixed

**Pull request merged**